src/test/resources/json/after-step-fails.json

```diff
@@ -0,0 +1,38 @@
+[
+    {
+        "id":"simpleId",
+        "name":"Simple feature",
+        "keyword":"Feature",
+        "elements":[
+            {
+                "keyword":"Scenario",
```

Comment:
For simplicity, I collapsed all the failure cases into a single JSON. If I put everything into sample.json, a lot of tests fail and it becomes a chore to update every other test. This seems like a happy medium.